test/testdata/lsp/highlight_untyped_typed_true.rb

```diff
@@ -0,0 +1,94 @@
+# typed: true
+
+# highlight-untyped-values: true
+
+extend T::Sig
+
+sig { returns(Integer) }
+def foo
+  my_map = T.let({ foo: 1, bar: 'baz' }, T::Hash[Symbol, T.untyped])
+  my_map[:foo]
+# ^^^^^^^^^^^^ info: This code is untyped
+end
+
+sig { params(x: Integer, y: String).returns(Integer) }
+def bar(x, y)
+	if x > y.length
+		x
+	else
+		y.length
+	end
+end
+
+sig { returns(T.untyped) }
+def baz
+	T.let(5, T.untyped)
+end
+
+# assign untyped thing to variable
+b = baz
+#   ^^^ info: This code is untyped
+
+# use an untyped variable
+  b.length
+# ^^^^^^^^ info: This code is untyped
+T.let(b, Integer) == 6
+
+
+my_map = T.let({:foo => 5, :bar => "foo"}, T::Hash[Symbol, T.untyped])
+# untyped argument
+bar(my_map[:foo], T.let("foo", T.untyped))
+#   ^^^^^^^^^^^^ info: This code is untyped
+
+# if condition
+if my_map[:foo]
+#  ^^^^^^^^^^^^ info: This code is untyped
+	6
+end
+
+# case statement
+case my_map[:bar]
+#    ^^^^^^^^^^^^ info: This code is untyped
+when "x"
+	"x"
+when "y"
+	"y"
+end
+
+# use of super
+class Base
+	extend T::Sig
+
+	sig { overridable.returns(String) }
+	def foo
+		"foo"
+	end
+end
+
+class Derived < Base
+	extend T::Sig
+	sig { override.returns(String) }
+	def foo
+		super
+# ^^^^^ info: This code is untyped
+	end
+end
+
+# untyped varargs
+sig { params(args: T.untyped).void }
+def args_fn(*args)
+	args.length
+end
+
+# untyped kwargs
+sig { params(kwargs: T.untyped).void }
+def kwargs_fn(**kwargs)
+	kwargs.keys
+end
+
+# use of &blk with untyped blk
+sig {params(blk: T.untyped).returns(T.untyped)}
+def blk_fun(&blk)
+	yield "x"
+#^^^^^^^^^ info: This code is untyped
+end
```

Comment:
@jez did you mean something more like

```ruby
sig { params(args: Integer).returns(Integer) }
def args_fn(*args)
  args.length
end

# untyped kwargs
sig { params(kwargs: Integer).returns(T::Array[Integer]) }
def kwargs_fn(**kwargs)
  kwargs.values
end

# use of &blk with untyped blk
sig {params(blk: T.proc.params(x: Integer).returns(Integer)).returns(Integer)}
def blk_fun(&blk)
  yield 5
end 

args_fn(b, b)
kwargs_fn(x: b, y: b)
blk_fun {|x| b}
```

because nothing gets reported for this (here, `b: T.untyped`)

Reply:
```ruby
sig { params(x: Integer, y: String).returns(Integer) }
def binary_method(x, y)
end

args = T::Array[T.untyped].new
binary_method(*args)
```

Reply:
Can't get this to work

Sorbet complains about `untyped_array_args` not being statically sized

If I make it `[T.let(1, T.untyped), T.let(1, T.untyped)]` it just doesn't complain about `untyped`.

With the `T.unsafe` one it complains about T.untyped on
```ruby
binary_method(*untyped_args)
^^^^^^^^^^^^^^^^^^^^^^^ untyped: This code is untyped
```
but also
````ruby
binary_method(*untyped_args)
  untyped: This code is untyped
```
That is, in the latter one doesn't seem to be pointing at any loc? not sure what's going on there?

Reply:
> doesn't seem to be pointing at any loc

yep, this is what I expected, we'll have to fix it in a future change 🙂

Reply:
> just doesn't complain about

this will be part of Phase 2. You can add the test now if you want.

> Sorbet complains about untyped_array_args not being statically sized

good point, forgot about that. feel free to leave that out of the test